types/ember-data__adapter/error.d.ts

```diff
@@ -0,0 +1,31 @@
+import DS from 'ember-data';
+
+export const AdapterError: typeof DS.AdapterError;
```

Comment:
proposal:
```suggestion
export import AdapterError = DS.AdapterError;
```

reexports all meanings of the thing, which is your intent here, right?

Reply:
Huh. I had no idea that was a thing; yes indeed that captures our intent. I'm happy to update all of these accordingly!